### ISSUE TITLE
Omit quotes around body

### ISSUE THREAD

**Issue by dandv** [kahmali/meteor-restivus#32]
I want to return a raw string as the [response data](https://github.com/kahmali/meteor-restivus#response-data) but restivus wraps the string in double quotes.

    return 'foo"bar'

Will produce

    "foo\"bar"

I've tried setting the Content-type to text/plain or text/html but I get the same type of output.

What I ultimately want to do is return a JPEG image from a custom `collection/:id/photo` route that extracts the base64-encoded image from a document.

**Comment by kahmali** [comment]
I'm looking into this. I'll keep you posted. If you have any suggestions just let me know.

**Comment by kahmali** [comment]
I'm pretty sure I know what's causing this. I was naively `JSON.stringify`ing the response, regardless of the Content-Type. I just need to find a way to consistently pass a `String` or `Buffer` to `response.write()`  in my `Route._respond()` method, while maintaining the proper content type. I've included the offending code below:

```coffeescript
  ###
    Respond to an HTTP request
  ###
  _respond: (endpointContext, body, statusCode=200, headers) ->
    # Allow cross-domain requests to be made from the browser
    endpointContext.response.setHeader 'Access-Control-Allow-Origin', '*'

    # Ensure that a content type is set (will be overridden if also included in given headers)
    # TODO: Consider enforcing a text/json-only content type (override any user-defined content-type)
    endpointContext.response.setHeader 'Content-Type', 'text/json'

    ##################################################################
    # BUG: Only JSON Content-Type responses should be JSON.stringified
    ##################################################################
    # Prettify JSON if configured in API
    if @api.config.prettyJson
      bodyAsJson = JSON.stringify body, undefined, 2
    else
      bodyAsJson = JSON.stringify body

    # Send response
    endpointContext.response.writeHead statusCode, headers
    endpointContext.response.write bodyAsJson
    endpointContext.response.end()
```

**Comment by kahmali** [comment]
The commit referenced above should fix this issue. I added a [basic Tinytest](https://github.com/kahmali/meteor-restivus/commit/4b30919e38af625e9e48b60c7c89624ea65e1553#diff-3eaed807d9ff886153a0f7fbbedffd73R207) and it passed that test along with the rest of the suite, so everything looks good to go from my end. Would you like to test it out on your use case before I bump the version and publish this?

**Comment by kahmali** [comment]
I went ahead and merged this fix into `master` since it continued to pass all my tests. It has been published in [v0.6.5](https://github.com/kahmali/meteor-restivus/compare/v0.6.4...v0.6.5). Please let me know if it doesn't behave as expected now.

**Comment by dandv** [comment]
Thanks @kahmali. Tests pass here as well. I've used "text/plain" and "text/html".
